Yosemite/YosemiteTests/Mockups/MockupNetwork+Path.swift

```diff
@@ -0,0 +1,22 @@
+import XCTest
+@testable import Networking
+
+extension MockupNetwork {
+    var pathComponents: [String]? {
```

Comment:
It can be good to add just a small comment which explains the meaning of `pathComponents` and the use case.

Reply:
yup good point! added in https://github.com/woocommerce/woocommerce-ios/pull/2209/commits/535a85ee1a77b84d9b0e1bb5da66bebed9695379